### DISCUSSION TITLE
ETA of next release? (0.8?)

### DISCUSSION THREAD

**Discussion by famod** [quarkiverse/quarkus-cxf#235] [General]
Hi all!

Any plans on cutting  a new release with that nice client enhancement?

Cheers,
Falko

**Comment by shumonsharif** [comment]
Hey @famod I believe @dufoli was planning on releasing after 1.13.5.Final is out!  I believe that should be any day now ...

https://github.com/quarkusio/quarkus/milestone/126?closed=1

**Reply by famod** [reply]
Ah yes, I do now remember him saying this. 1.13.5.Final is already on Maven central btw, but the release page is not fully published and there were some github doc publish issues earlier today.
https://github.com/quarkusio/quarkus/releases/tag/1.13.5.Final

**Comment by dufoli** [comment]
I will right now.
